plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -43,6 +43,7 @@ local TextBoxWidget = require("ui/widget/textboxwidget")
 local TitleBar = require("ui/widget/titlebar")
 local ToggleSwitch = require("ui/widget/toggleswitch")
 local UIManager = require("ui/uimanager")
+local util = require("util")
```

Comment:
Could you move it under WidgetContainer above _, see https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires

Reply:
Done.